Assets/Gameplay/Behaviours/AttackBehaviour.cs

```diff
@@ -24,6 +24,45 @@ public class AttackBehaviour : BaseBehaviour
         bool HasValidTarget => CurrentTarget != null;
 
         float _attackTimer = 0;
+        public bool isAreaAttack = false;
+
+        public float distanceSplashDamage = 3;
+        private List<IDamageable> getEnemies()
+        {
+            var myTeam = Entity.Team;
+            var enemyTeam = "";
```

Comment:
Aqui a gente pode usar um [método de extensão](https://docs.microsoft.com/pt-br/dotnet/csharp/programming-guide/classes-and-structs/extension-methods) do enum Team que já tá implementado no `Team.cs`. Basta chamar `myTeam.Opposite()` que este método retornará o time oposto.